slapp/transfers/upload.py

```diff
@@ -38,26 +41,42 @@ def run(self, db_conn: query_utils.DbConnection):
         query_string = ("SELECT manifest FROM roi_manifests "
                         f"WHERE id in ({idstr})")
         manifests = [r['manifest'] for r in db_conn.query(query_string)]
+        nman = len(manifests)
+        self.logger.info(
```

Comment:
We might want to log a warning if they don't agree on length. I'm okay with it either way just a thought for a more verbose log message.

Reply:
Good idea, added that and updated the example output above.

Reply:
Looks good to me!